pages/docs/index.mdx

```diff
@@ -26,6 +26,8 @@ Drei Gründe, die für *Stadtpuls* sprechen:
 <br></br>
 <span className="w-12 h-12 rounded-full bg-gray-200" style={{padding: '0.3em 0.5em 0.3em 0.5em'}}>→</span> Die Nutzung der Plattform ist komplett <strong>kostenlos</strong>
 
+&#x26a0; Achtung: Stadtpuls befindet sich noch in einer frühen Entwicklungsphase, daher können noch Fehler auftreten und es an verschiedenen Stellen haken. Wir bitten euch daher um Verständnis und ein wenig Geduld. Wenn du Fragen, Feedback oder Anregungen hast, dann schreib uns eine E-Mail an: *info@citylab-berlin.org* oder nutze <a href="https://github.com/technologiestiftung/stadtpuls-community/discussions">Github Discussions</a>, um dich mit uns und der Community zu auszutauschen. Bugs kannst Du als <a href="https://github.com/technologiestiftung/stadtpuls-community/issues">Github Issue</a> reporten.
```

Comment:
Maybe instead of the Unicode symbol, we can use our custom warning icon to ensure a consistent display across clients?

It is located at `public/images/icons/16px/warningDuotone.svg` and can be used in an `<img>` tag.

Reply:
good catch! I corrected the wording accordingly, thanks.
I kept the ascii code instead of the icon as styling this icon proved a bit too time consuming and I considered it not worth it right now (we use the code elsewhere, too)